### DISCUSSION TITLE
How to Acess UI interaction in Truffle for VS Code

### DISCUSSION THREAD

**Discussion by Harsh30199** [ConsenSys-archive/truffle#5321] [VSCode Extension] [answered]
Hi

I was trying to use Truffle for VS Code extension and am stuck at how to access the UI interaction mentioned in truffle extension docs.

Link to docs :  https://trufflesuite.com/docs/vscode-ext/debugging-smart-contracts/

Screenshot:
![Screenshot from 2022-07-19 01-39-20](https://user-images.githubusercontent.com/38328698/179608787-8892b661-402b-4ebd-b5bf-e7e4ac68c470.png)

On clicking UI Interaction hyperlink it gives 404 not found

**Comment by michaeljohnbennett** [comment, marked as answer]
Hi @Harsh30199, thanks for raising this, yes this is well spotted!

We will amend the docs and I think this feature wasn't that great in the code originally so was removed a while ago but the docs didn't get updated as a result.

There are more ways to interact with your contract coming in the future across the truffle ecosystem but if you wanted to interact with your contract right now the truffle console would be your best bet.

You can get some docs here on this: https://trufflesuite.com/docs/truffle/getting-started/interacting-with-your-contracts/

**Comment by jmseigneur** [comment]
It is a pity that there is no "UI interaction" anymore. It seems that the "Ethereum Remix" extension may help to have a quick UI but there are some issues to detect the transactions and calls made with this extension on the same Ganache as used by the Truffle VS plugin, which doesn't find the code of the executed transactions and calls in the debugger.
